_topic_map.yml

```diff
@@ -122,6 +122,8 @@ Name: Installing
 Dir: installing
 Distros: openshift-origin,openshift-enterprise,openshift-webscale
 Topics:
+- Name: Installation overview
```

Comment:
For reference, left a comment about this title: https://github.com/openshift/openshift-docs/commit/2092035c74d685dff7b072f5660300985c6f5f4f#r48283647